frontend/public/components/dashboard/dashboards-page/overview-dashboard/queries.ts

```diff
@@ -17,9 +17,8 @@ export enum OverviewQuery {
 }
 
 const overviewQueries = {
-  [OverviewQuery.MEMORY_TOTAL]: 'sum(kube_node_status_capacity_memory_bytes)',
-  [OverviewQuery.MEMORY_UTILIZATION]:
-    '(sum(kube_node_status_capacity_memory_bytes) - sum(kube_node_status_allocatable_memory_bytes))[60m:5m]',
+  [OverviewQuery.MEMORY_TOTAL]: 'sum(node_memory_MemTotal_bytes)',
+  [OverviewQuery.MEMORY_UTILIZATION]: 'sum(node_memory_Active_bytes)',
```

Comment:
Unfortunately, converting this to an instant query breaks the memory graph on the dashboard's "Cluster Utilization" card, so I think we need to put it back to a range query for now.

Those dashboard graphs need to be changed to use `query_range` instead of `query`. (Also discussed in https://github.com/openshift/console/pull/2585#issuecomment-528292025.)

CC @rawagner

Reply:
after we merge https://github.com/openshift/console/pull/2988, this change will be ok - wont break the graph anymore

Reply:
👍 Thanks @rawagner